llvm/lib/InterfaceStub/IFSHandler.cpp

```diff
@@ -201,6 +201,12 @@ Expected<std::unique_ptr<IFSStub>> ifs::readIFSFromBuffer(StringRef Buf) {
           "IFS arch '" + *Stub->Target.ArchString + "' is unsupported");
     Stub->Target.Arch = eMachine;
   }
+  for (const auto &item : Stub->Symbols) {
+    if (item.Type == IFSSymbolType::Unknown)
+      return createStringError(
+          std::make_error_code(std::errc::invalid_argument),
+          "IFS symbol type for symbol '" + item.Name + "' is unsupported");
```

Comment:
This is just a suggestion, but I'd use `"unsupported type for symbol '" + item.Name + "'"`.

Reply:
The error message I used here matches the error message used https://github.com/llvm/llvm-project/pull/75872/files/3374f89d95e7888e0edb01a205cb2b4c85185265#diff-2d9e2d92eeaa1f78b0ff93f60f50e82c16f9adc6ef19768ef750e5c888c6f32eR201
I can create a separate PR to change the error messages of those cases if you prefer.

Reply:
It might also be useful to include what the actual type is, if practical.

Reply:
Unfortunately after YAML parsing, the original type string will not be recorded, so it is not practical right now. Unless we refractor the code to use string to store the Symbol type information and parsed it out side of LLVM YAML I/O, the type name cannot be present in the error message here.

Reply:
I agree about making the error message consistent (and changing them in a follow up PR).